### PR TITLE
Bay clicky noises 3: Zzzzzip

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

Doesn't actually add much clicky noises besides:
Radios and intercoms make clicking noises when toggling speakers and microphone

...instead adds sounds whenever you open a storage item including but not limited to:
Boxes
Backpacks
Briefcases
Tooboxes
Pill bottles

In additions, adds pickup and dropping items for many many tools

Demodularizes firstaid.dmi

Sounds from
https://github.com/NebulaSS13/Nebula/commit/30a68bebfa44ce562582171c8d25c4ceb0740ad5
https://github.com/NebulaSS13/Nebula/commit/9c69c006d7dd950479391d58a50e6d8eae6f9ce0
https://github.com/NebulaSS13/Nebula/pull/1478

## Why It's Good For The Game

i fucking love sound design

## Changelog
:cl:
soundadd: Intercoms and radios now will click when toggling buttons
soundadd: Storage items will have different sounds depening on what they are opening
soundadd: A TON of sounds added when picking up and dropping things
/:cl:

### CODE REVIEW

File: code/datums/components/storage/storage.dm [hunk 1]
```diff
@@ -578,6 +578,8 @@
 	// this must come before the screen objects only block, dunno why it wasn't before
 	if(over_object == M)
 		user_show_to_mob(M)
+		if(use_sound)
```
Comment (rye-rice) [inline code/datums/components/storage/storage.dm:581]:
```suggestion
		if(use_sound) 
```

File: code/game/objects/items/storage/firstaid.dm [hunk 2]
```diff
@@ -8,20 +8,25 @@
...
+	pickup_sound =  'sound/items/handling/cardboardbox_pickup.ogg'
 	throw_speed = 3
 	throw_range = 7
 	var/empty = FALSE
 	var/damagetype_healed //defines damage type of the medkit. General ones stay null. Used for medibot healing bonuses
 
+/obj/item/storage/firstaid/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.use_sound = 'sound/items/storage/briefcase.ogg'
```
Comment (Fikou) [inline code/game/objects/items/storage/firstaid.dm:28]: componentinitialize is deprecated, use initialize, same for all the other ones

File: code/datums/components/storage/storage.dm [hunk 3]
```diff
@@ -578,6 +578,8 @@
...
```
Comment (rye-rice) [inline code/datums/components/storage/storage.dm]:
```suggestion
		if(use_sound)
```

File: code/game/objects/items/storage/firstaid.dm (hunk 2 above)
Comment (Fikou) [reply code/game/objects/items/storage/firstaid.dm:28]: nvm we use it for storage thats stupid

**Pull request merged**